tests/Common/TokensTest.php

```diff
@@ -1336,9 +1338,6 @@ public function testReadOnlyRoleBucketsPermissions(): void
         foreach ($token['bucketPermissions'] as $bucketId => $permission) {
             $this->assertEquals('read', $permission);
         }
-
-        $client->getBucket($newBucketId);
```

Comment:
Tohle by se nemělo mazat, to je ten test, ne? Že to nespadne na exceptioně...

Reply:
vratil som to